controller/src/main/java/io/pravega/controller/store/stream/InMemoryStream.java

```diff
@@ -702,6 +704,55 @@ private void handleStreamMetadataExists(final long timestamp, CompletableFuture<
         return result;
     }
 
+    @Override
+    CompletableFuture<Void> createSealedSegmentsRecord(byte[] sealedSegmentRecord) {
+        Preconditions.checkNotNull(sealedSegmentRecord);
+
+        CompletableFuture<Void> result = new CompletableFuture<>();
```

Comment:
No need for an explicit `result` variable. Just return `CompletableFuture.completedFuture(null)`

Reply:
done